### PR TITLE
Interactive approach: change known predicates setting to string format

### PR DESCRIPTION
**Pull request by amburger66**
Changes the `interactive_known_predicates` config setting to be a string, so it can easily be changed from the commandline.

### CODE REVIEW

File: tests/approaches/test_interactive_approach.py [hunk 1]
```diff
@@ -55,18 +55,30 @@ def test_create_teacher_dataset():
...
-        "timeout": 10,
-        "max_samples_per_step": 10,
-        "seed": 12345,
-        "sampler_mlp_classifier_max_itr": 500,
-        "predicate_mlp_classifier_max_itr": 500,
-        "neural_gaus_regressor_max_itr": 500,
-        "interactive_num_episodes": 1,
-        "interactive_relearn_every": 1,
-        "interactive_known_predicates": {'HandEmpty', 'Holding', 'Covers'}
+        "env":
```
Comment (tomsilver) [inline tests/approaches/test_interactive_approach.py:58]: lol oh gosh, did the autoformatter do this?

Comment (tomsilver) [reply tests/approaches/test_interactive_approach.py:58]: I guess the autoformatter is boss now...

**Review by tomsilver — APPROVED** [review]
lgtm

File: tests/approaches/test_interactive_approach.py (hunk 1 above)
Comment (amburger66) [reply tests/approaches/test_interactive_approach.py:58]: yeah....

**Pull request merged**